plugins/lacework/api_key_test.go

```diff
@@ -0,0 +1,71 @@
+package lacework
+
+import (
+	"testing"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func TestAPIKeyImporter(t *testing.T) {
+	plugintest.TestImporter(t, APIKey().Importer, map[string]plugintest.ImportCase{
+		"environment": {
+			Environment: map[string]string{
+				"LW_ACCOUNT":    "example",
+				"LW_API_KEY":    "EXAMPLE_1234567890ABCDE1EXAMPLE1EXAMPLE123456789EXAMPLE",
+				"LW_API_SECRET": "_aaaaaaaaaaaaaaaaaaaaaaaaaexample",
+			},
+			ExpectedCandidates: []sdk.ImportCandidate{
+				{
+					Fields: map[sdk.FieldName]string{
+						fieldname.Account:   "example",
+						fieldname.APIKey:    "EXAMPLE_1234567890ABCDE1EXAMPLE1EXAMPLE123456789EXAMPLE",
+						fieldname.APISecret: "_aaaaaaaaaaaaaaaaaaaaaaaaaexample",
+					},
+				},
+			},
+		},
+		"Lacework config file": {
+			Files: map[string]string{
+				"~/.lacework.toml": plugintest.LoadFixture(t, "lacework.toml"),
+			},
+			ExpectedCandidates: []sdk.ImportCandidate{
+				{
+					Fields: map[sdk.FieldName]string{
+						fieldname.Account:   "example",
+						fieldname.APIKey:    "EXAMPLE_1234567890ABCDE1EXAMPLE1EXAMPLE123456789EXAMPLE",
+						fieldname.APISecret: "_aaaaaaaaaaaaaaaaaaaaaaaaaexample",
+					},
+				},
+				{
+					Fields: map[sdk.FieldName]string{
+						fieldname.Account:   "example2",
+						fieldname.APIKey:    "EXAMPLE2_1234567890ABCDE1EXAMPLE1EXAMPLE123456789EXAMPLE",
+						fieldname.APISecret: "_bbbbbbbbbbbbbbbbbbbbbbbbbexample",
+					},
+					NameHint: "example",
+				},
+			},
+		},
+	})
+}
+
+func TestAPIKeyProvisioner(t *testing.T) {
+	plugintest.TestProvisioner(t, APIKey().DefaultProvisioner, map[string]plugintest.ProvisionCase{
+		"default": {
+			ItemFields: map[sdk.FieldName]string{
+				fieldname.Account:   "example",
+				fieldname.APIKey:    "EXAMPLE_1234567890ABCDE1EXAMPLE1EXAMPLE123456789EXAMPLE",
+				fieldname.APISecret: "_aaaaaaaaaaaaaaaaaaaaaaaaaexample",
+			},
+			ExpectedOutput: sdk.ProvisionOutput{
+				Environment: map[string]string{
+					"LW_ACCOUNT":    "example",
+					"LW_API_KEY":    "EXAMPLE_1234567890ABCDE1EXAMPLE1EXAMPLE123456789EXAMPLE",
+					"LW_API_SECRET": "_aaaaaaaaaaaaaaaaaaaaaaaaaexample",
```

Comment:
Would be nice to use more realistic values here. We have a Makefile command to easily generate those:
```
make lacework/example-secrets
```

Reply:
This works, in part, for the API secret since it has a fixed/known length (32 hex characters, prefixed with `_`), but this isn't true for the other two fields. I've taken the API key (ID) value from [here](https://docs.lacework.com/cli/#configure-the-cli), but the prefix before the `_` is variable so the total length isn't deterministic.

For the secret, I've got an "expected" format, with the exception of the `example` suffix not be hex, but I was following the pattern that `%/example-secrets` produces to try to be consistent.

```
➜  shell-plugins git:(lacework) make lacework/example-secrets
go run cmd/contrib/main.go lacework/example-secrets
API Key:
  Account:
  API Key:
  API Secret: _6wz23kv3ry45eg78xwd6m6_vqexample
```

Is there another way to handle this within the current framework?

Reply:
I think the account and API key values you're using now are fine. If you can update the API secret test values with this example value then we should be good to go!

Reply:
Perfect, thanks! What do you think about using something like this instead, so that the example is more correct/representative? The `_` in the middle of the previous example isn't quite correct since it's only used as a prefix (nor are some of the characters in `example`, but I think that's clear enough).

```
➜  shell-plugins git:(lacework) echo -n '_'; openssl rand -hex 16 | cut -c8- | tr -d '\n'; echo 'example'
_72a101a5e009abc47c32c6ff9example
```

Reply:
Yep, looks good to me. And FYI, if the `_` always is the prefix, you can add `Prefix: "_"` to the struct.

Reply:
Great, thank you. I've got that there already and I also included `Specific:  []rune{'_'}` in the charset. I assumed this was required since it was used in the prefix, but perhaps it's not?

Reply:
Ah right, then you don't need the `Specific` entry.

Reply:
Will remove that and push the newly generated secrets too.